src/bucket/BucketTests.cpp

```diff
@@ -1109,36 +1109,60 @@ TEST_CASE("bucket apply", "[bucket]")
     REQUIRE(count == 1);
 }
 
-#ifdef USE_POSTGRES
 TEST_CASE("bucket apply bench", "[bucketbench][!hide]")
 {
-    VirtualClock clock;
-    Config cfg(getTestConfig(0, Config::TESTDB_POSTGRESQL));
-    Application::pointer app = createTestApplication(clock, cfg);
-    app->start();
+    auto runtest = [](Config::TestDbMode mode) {
+        VirtualClock clock;
+        Config cfg(getTestConfig(0, mode));
+        Application::pointer app = createTestApplication(clock, cfg);
+        app->start();
 
-    std::vector<LedgerEntry> live(100000);
-    std::vector<LedgerKey> noDead;
+        std::vector<LedgerEntry> live(100000);
+        std::vector<LedgerKey> noDead;
 
-    for (auto& l : live)
-    {
-        l.data.type(ACCOUNT);
-        auto& a = l.data.account();
-        a = LedgerTestUtils::generateValidAccountEntry(5);
-    }
+        for (auto& l : live)
```

Comment:
Why loop over `LedgerTestUtils::generateValidAccountEntry` instead of directly using `LedgerTestUtils::generateValidAccountEntries`?

Reply:
I didn't write this code but it seems that it would require generating an array of AccountEntry and then transform it into an array of LedgerEntry.